crates/transaction-emitter-lib/src/emitter/account_minter.rs

```diff
@@ -203,7 +203,7 @@ impl<'t> AccountMinter<'t> {
                 )
             });
 
-        // Each future creates 50 accounts, limit concurrency to 30.
+        // Each future creates 10 accounts, limit concurrency to 100.
```

Comment:
This function seems fairly complex, do we have unitesting for it?

Reply:
yeah, it doesn't. but minter is really a testing utility, so not really clear how much is it worth spending on testing a test code